annotator-core/src/main/java/edu/ucr/cs/riple/core/metadata/trackers/generatedcode/LombokTracker.java

```diff
@@ -0,0 +1,41 @@
+package edu.ucr.cs.riple.core.metadata.trackers.generatedcode;
+
+import edu.ucr.cs.riple.core.metadata.method.MethodDeclarationTree;
+import edu.ucr.cs.riple.core.metadata.trackers.MethodRegionTracker;
+import edu.ucr.cs.riple.core.metadata.trackers.Region;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class LombokTracker implements GeneratedRegionTracker {
```

Comment:
Let's add some Javadoc on this class that documents what our assumptions are in terms of how exactly Lombok propagates annotations, so we know what we are trying to handle.

Reply:
f0880e2067988c53d64832202e269c34e7366588